CMakeLists.txt

```diff
@@ -81,14 +81,20 @@ if(DJINNI_WITH_OBJC)
 endif()
 
 # JNI support
-option(DJINNI_WITH_JNI "Include the JNI support code in Djinni support library." off)
+option(DJINNI_WITH_JNI "Include the JNI support code in Djinni support library." OFF)
+
+option (JNI_CPP_THREAD_ATTACH "Add code enableing calling from C++ thread to Java (experimatal)" OFF)
```

Comment:
```suggestion
option (JNI_CPP_THREAD_ATTACH "Add code enableing calling from C++ thread to Java (experimental)" OFF)
```

Reply:
we have other `off` also :-) 

but we can start here with the to `OFF` transformation :-)